Doc/library/venv.rst

```diff
@@ -36,54 +47,66 @@ Creating virtual environments
-   operating system.
-
-   A virtual environment is a directory tree which contains Python executable
-   files and other files which indicate that it is a virtual environment.
-
-   Common installation tools such as setuptools_ and pip_ work as
-   expected with virtual environments. In other words, when a virtual
-   environment is active, they install Python packages into the virtual
-   environment without needing to be told to do so explicitly.
-
-   When a virtual environment is active (i.e., the virtual environment's Python
-   interpreter is running), the attributes :attr:`sys.prefix` and
-   :attr:`sys.exec_prefix` point to the base directory of the virtual
-   environment, whereas :attr:`sys.base_prefix` and
-   :attr:`sys.base_exec_prefix` point to the non-virtual environment Python
-   installation which was used to create the virtual environment. If a virtual
-   environment is not active, then :attr:`sys.prefix` is the same as
-   :attr:`sys.base_prefix` and :attr:`sys.exec_prefix` is the same as
-   :attr:`sys.base_exec_prefix` (they all point to a non-virtual environment
-   Python installation).
-
-   When a virtual environment is active, any options that change the
-   installation path will be ignored from all ``setuptools`` configuration
-   files to prevent projects being inadvertently installed outside of the
-   virtual environment.
-
-   When working in a command shell, users can make a virtual environment active
-   by running an ``activate`` script in the virtual environment's executables
-   directory (the precise filename and command to use the file is
-   shell-dependent), which prepends the virtual environment's directory for
-   executables to the ``PATH`` environment variable for the running shell. There
-   should be no need in other circumstances to activate a virtual
-   environment; scripts installed into virtual environments have a "shebang"
-   line which points to the virtual environment's Python interpreter. This means
-   that the script will run with that interpreter regardless of the value of
-   ``PATH``. On Windows, "shebang" line processing is supported if you have the
-   Python Launcher for Windows installed (this was added to Python in 3.3 - see
-   :pep:`397` for more details). Thus, double-clicking an installed script in a
-   Windows Explorer window should run the script with the correct interpreter
-   without there needing to be any reference to its virtual environment in
-   ``PATH``.
+Further venv detail
+-------------------
+
+When a virtual environment is running, the attributes :attr:`sys.prefix` and
+:attr:`sys.exec_prefix` point to the prefix directory of the virtual
+environment, whereas :attr:`sys.base_prefix` and
+:attr:`sys.base_exec_prefix` point to the non-virtual environment Python
+installation which was used to create the virtual environment (known as the
+virtual environment's base environment). It is sufficient to check
```

Comment:
I find the term "active" to be confusing - it could be understood that this is only "active" after it has been "activated", which of course is not correct.

I accept that the term running is also not entirely natural. I don't have very strong proposals though. The best I could come up with:

> When Python is running from a virtual environment, ...

Reply:
I can see your concern and that this could be more precise and specific (though "active" is still much more appropriate than "running", given the former is standard—if slightly imprecise—verbiage). The clear solution appears to be simply stating that the Python interpreter in question is running in a virtual environment, just as you and @vsajip have suggested. I've modified my suggestion above accordingly.